### PR TITLE
Setup CI

### PR DESCRIPTION
**Pull request by HoshinoTented**
Use refined intellij-platform-plugin-template.

### CODE REVIEW

File: src/main/kotlin/org/ice1000/kala/utils.kt [hunk 1]
```diff
@@ -20,8 +20,7 @@ class KalaInspectionProvider : InspectionToolProvider {
     ViewSizeInspection::class.java,
     SizeCompareInspection::class.java,
     MapPutUnetaInspection::class.java,
-    ViewToMapInspection::class.java,
-    SamenessInspection::class.java
```
Comment (ice1000) [inline src/main/kotlin/org/ice1000/kala/utils.kt:24]: Why did you delete this?

Comment (HoshinoTented) [reply src/main/kotlin/org/ice1000/kala/utils.kt:24]: Because there is no `SamenessInspection.kt`, this code is introduced by #2 in mistake.
